packages/playwright-core/src/server/injected/roleUtils.ts

```diff
@@ -461,6 +461,35 @@ export function getElementAccessibleDescription(element: Element, includeHidden:
   return accessibleDescription;
 }
 
+export function getElementAccessibleErrorMessage(element: Element): string {
+  const cache = cacheAccessibleErrorMessage;
+  let accessibleErrorMessage = cacheAccessibleErrorMessage?.get(element);
+
+  if (accessibleErrorMessage === undefined) {
+    accessibleErrorMessage = '';
+
+    const ariaInvalid = element.getAttribute('aria-invalid');
+    const isAriaInvalid = ariaInvalid !== null && ariaInvalid.toLowerCase() !== 'false';
+    if (isAriaInvalid) {
+      const errorMessageId = element.getAttribute('aria-errormessage');
+      if (errorMessageId) {
+        const errorMessages = getIdRefs(element, errorMessageId);
+        if (errorMessages.length) {
```

Comment:
nit: this `if` is unnecessary, joining over an empty array will be an empty string.